lib/stitch_fix/log_weasel/middleware.rb

```diff
@@ -1,6 +1,7 @@
 module StitchFix
   class LogWeasel::Middleware
     KEY_HEADER = 'X_LOGWEASEL_KEY'
+    CORRELATION_ID_ENV_KEY = "HTTP_X_CORRELATION_ID"
```

Comment:
Naming nit, but what do you think of `CORRELATION_ID_HEADER` or `CORRELATION_ID_HEADER_KEY`?

Reply:
I had started with `CORRELATION_ID_HEADER_KEY` but realized that `HTTP_X_CORRELATION_ID` is a rack or rails env value populated from the request headers, IIUC. So I canged to `...ENV_KEY`. I am fine with `CORRELATION_ID_HEADER_KEY` or even `CORRELATION_ID_KEY` as that is really what we are after.

I'll update Real Soon Now™ (RSN).